public/index.html

```diff
@@ -1,15 +1,19 @@
 <!DOCTYPE html>
-<html lang="en" >
-<head>
-  <meta charset="UTF-8">
-  <title>ayaqdop</title>
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta name="author" content="Olzhas Askar">
-  <meta name="description" content="Chess-like football game."/>
-  <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Overpass'>
-</head>
-<body>
-  <div id="app"></div>
-</body>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>ayaqdop</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="author" content="Olzhas Askar" />
+    <meta name="description" content="Chess-like football game." />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Overpass"
+    />
+    <base href="/" />
```

Comment:
just curious, what does this tag do?

Reply:
it was impossible to import images in dynamic react-router-dom paths like `match\:matchId`